index.js

```diff
@@ -0,0 +1,22 @@
+const tmp = require('tmp');
```

Comment:
Worth considering, using `esm` with node so all source files are using the module specification.

Reply:
done